rewrite-maven/src/main/java/org/openrewrite/maven/UncheckedMavenDownloadingException.java

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright 2020 the original author or authors.
- * <p>
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * <p>
- * https://www.apache.org/licenses/LICENSE-2.0
- * <p>
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.openrewrite.maven;
-
-import lombok.Getter;
-import org.openrewrite.xml.tree.Xml;
-
-@Getter
-public class UncheckedMavenDownloadingException extends RuntimeException {
```

Comment:
Usage seems to have been dropped in https://github.com/openrewrite/rewrite/commit/6886deec23dcc1df31f121fd4545a89e9df3e27e